### PR TITLE
ci update for travis

### PR DESCRIPTION
**Pull request by austinpapp**

### CODE REVIEW

**Comment by austinpapp** [comment]
@igaponov just take a look and make sure this LGTM (you).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: php
+php:
+  - 5.5
+before_script:
+  - phpenv config-rm xdebug.ini; true
+  - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - cd backend
+  - cp app/config/parameters.travis.yml app/config/parameters.yml
+  - composer install --ignore-platform-reqs --no-interaction --prefer-source
```
Comment (igaponov) [inline .travis.yml]: --prefer-dist can be faster I think.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - php app/console doctrine:database:create --env=test
+  - php app/console doctrine:schema:create --env=test
```
Comment (igaponov) [inline .travis.yml]: not necessary for test, db is created every time for every test (or copy from generated prototype)

Comment (austinpapp) [inline .travis.yml]: ok. let me test with that to confirm that works as we expect. just so i understand, as long as the `parameters.yml` is accurate, the test framework will build the db as necessary ?

Comment (igaponov) [inline .travis.yml]: Yes, test bundle creates db for every test (with schema and migrations).

**Pull request merged**